src/commonmark/en/content/user/configure-metadata.md

```diff
@@ -748,7 +748,7 @@ objects:
     </tr>
     <tr class="even">
     <td><p>Username</p></td>
-    <td><p>DHIS2 user. Rendered as a dialog with a list of users and a search field.</p></td>
+    <td><p>DHIS2 user. Rendered as a dialog with a list of users and a search field. The user will need the "View User" authority to be     able to utilise this data type</p></td>
```

Comment:
remove the extra spaces between "be" and "able". 
In most cases this will render fine but markdown does treat spaces specially (for example, line breaks are ignored in a paragraph of text, unless the break is preceeded by two spaces - in which case it renders _with_ a line break).